src/platforms/PolygonId/components/MintableRole.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card, HStack, Heading, Spacer } from "@chakra-ui/react"
+import Button from "components/common/Button"
+import GuildLogo from "components/common/GuildLogo"
+import { Role } from "types"
+
+type Props = {
```

Comment:
We could simplify this component to have only 1 prop (`role`), see my other comments for more context ;)